apps/web/vibes/2px/components.ts

```diff
@@ -3,6 +3,13 @@ import { lazy } from 'react'
 import { Components } from '@/vibes/schema'
 
 export const components = [
+  {
+    name: 'accordions',
+    dependencies: ['@radix-ui/react-accordion'],
+    registryDependencies: [],
+    files: ['./components/accordions/index.tsx'],
```

Comment:
I think we need to include the icons file in here as well since it's used by the Accordions component

Reply:
I believe we missed this for all components. we'll make a PR to fix all components configs.